libs/ddhub-client-gateway-frontend/ui/roles/src/lib/models/roles-headers.tsx

```diff
@@ -2,29 +2,34 @@ import {
   TableHeader,
   CopyToClipboard,
 } from '@ddhub-client-gateway-frontend/ui/core';
-import { Box, Chip, chipClasses, Typography } from '@mui/material';
+import { Box, Chip, Typography } from '@mui/material';
 import { DateTime } from 'luxon';
 import { alpha, Theme } from '@mui/material/styles';
-import {
-  ExpirationStatus,
-  RoleStatus,
-} from '../components/RoleList/RoleList.types';
+import { ExpirationStatus } from '../components/RoleList/RoleList.types';
+import { RequesterClaimDTOStatus } from '@dsb-client-gateway/dsb-client-gateway-api-client';
+
+export enum RoleStatus {
```

Comment:
maybe you can rename this to `RoleStatusLabel` to avoid confusion with `RoleStatus` from '../components/RoleList/RoleList.types'